templates/paragraphs/paragraph--oe-description-list.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Description list' type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
```

Comment:
if we extend our template we could wrap this then in {% block paragraph %} and the content part in {% block content %} for anyone wanting to extend this further